cookbook/symfony1.rst

```diff
@@ -200,14 +200,14 @@ Applications
 ------------
 
 In a symfony1 project, it is common to have several applications: one for the
-frontend and one for the backend for instance.
+front-end and one for the back-end for instance.
```

Comment:
back-end and front-end seem to be the common English terms when looking at some dictionaries.